src/main/java/it/pagopa/transactions/exceptions/CheckoutRedirectMissingPspRequestedException.java

```diff
@@ -0,0 +1,26 @@
+package it.pagopa.transactions.exceptions;
+
+import java.util.Set;
+
+/**
+ * Exception thrown when requesting an API key from NPG configuration for a
+ * nonexisting PSP
+ */
+public class CheckoutRedirectMissingPspRequestedException extends RuntimeException {
```

Comment:
Please add exception handler method for this exception

Reply:
On second thought I would leave this to the PR implementing Checkout Redirect, if the exceptio is mapped to another exception then we wouldn't need a handler